spec/features/account_connected_apps_spec.rb

```diff
@@ -84,21 +91,27 @@
       click_link(t('help_text.requested_attributes.change_email_link'))
     end
 
-    expect(page).to have_field(user.email) { |field| !field[:checked] }
+    expect(page).to have_field(user.email) { |field| field[:checked] }
```

Comment:
Can you add an assertion around here of the value of the field we're expecting to be checked when they visit while not already having an email selected? (i.e. the most recent login email)

Reply:
Added that in [fa401c9](https://github.com/18F/identity-idp/pull/11568/commits/fa401c984bda1020a81b372709837fd9f3a9dd04)